src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.ts

```diff
@@ -230,9 +236,9 @@ export class DragAndDropQuestionComponent implements OnChanges {
      * (Only possible if this.question.correctMappings is available)
      *
      * @param dropLocation {object} the drop location to check for correctness
-     * @return {boolean} true, if the drop location is correct, otherwise false
+     * @return {boolean | undefined} true, if the drop location is correct, false if not and undefined if the location is correctly left blank
```

Comment:
I am not sure if we should add the new use case "correctly left blank" as undefined or if we should introduce an enum type `MappingResult` representing the 3 possible status `MAPPED_CORRECT`, `MAPPED_INCORRECT`, `NOT_MAPPED`. This would avoid comparing against undefined, which is not intuitively covering the use case of a dropLocation being correctly left blank.

Reply:
I agree that this makes the return value handling a bit prettier.